### PR TITLE
Added --verbose and --verbose-log-file to 'oscap ds rds-validate'.

### PR DESCRIPTION
**Pull request by rsprudencio**
Added --verbose and --verbose-log-file to 'oscap ds rds-validate'.

### CODE REVIEW

File: utils/oscap-ds.c [hunk 1]
```diff
@@ -241,12 +249,12 @@ bool getopt_ds(int argc, char **argv, struct oscap_action *action) {
 		action->ds_action->oval_result_count = argc - optind - 3;
 	}
 	else if( (action->module == &DS_RDS_VALIDATE_MODULE) ) {
-		if(  argc != 4 ) {
-			oscap_module_usage(action->module, stderr, "Wrong number of parameters.\n");
+		if(optind >= argc) {
+			oscap_module_usage(action->module, stderr, "Result DataStream file need to be specified!\n");
```
Comment (jan-cerny) [inline utils/oscap-ds.c]: Great! thanks

**Comment by jan-cerny** [comment]
Could you document this in the man page?

**Comment by jan-cerny** [comment]
process58 probe failing again. should be unrelated to this pull request.

**Comment by rsprudencio** [comment]
@openscap-jenkins test this please

**Comment by jan-cerny** [comment]
ACK

**Pull request merged**